src/waypoint.jsx

```diff
@@ -333,13 +352,18 @@ export default class Waypoint extends React.Component {
    * @return {Object}
    */
   render() {
+    if (this.props.children) {
+      return <div ref={this.refElement}>{this.props.children}</div>;
```

Comment:
Yup. I'll go ahead and build that then?

I'm planning to also allow users to pass an element without a ref, and have one injected using `cloneWithProps` or equivalent, like so:

```jsx
<Waypoint onEnter={this.handleWaypointEnter}>
  <div>
    Something interesting would go here.
  </div>
</Waypoint>
```

in which `div` ends up with a `ref` created/managed by Waypoint.